cmd/boost/wallet_cmd.go

```diff
@@ -132,42 +155,82 @@ var walletList = &cli.Command{
 					}
 				}
 
-				row := map[string]interface{}{
-					"Address": addr,
-					"Balance": types.FIL(a.Balance),
-					"Nonce":   a.Nonce,
+				wallet := map[string]interface{}{
+					addressKey: addr,
+					balanceKey: types.FIL(a.Balance),
+					nounceKey:  a.Nonce,
 				}
 				if addr == def {
-					row["Default"] = "X"
+					wallet[defaultKey] = "X"
 				}
 
 				if cctx.Bool("id") {
 					id, err := api.StateLookupID(ctx, addr, types.EmptyTSK)
 					if err != nil {
-						row["ID"] = "n/a"
+						wallet[idKey] = "n/a"
 					} else {
-						row["ID"] = id
+						wallet[idKey] = id
 					}
 				}
 
 				mbal, err := api.StateMarketBalance(ctx, addr, types.EmptyTSK)
 				if err == nil {
-					row["Market(Avail)"] = types.FIL(types.BigSub(mbal.Escrow, mbal.Locked))
-					row["Market(Locked)"] = types.FIL(mbal.Locked)
+					wallet[marketAvailKey] = types.FIL(types.BigSub(mbal.Escrow, mbal.Locked))
+					wallet[marketLockedKey] = types.FIL(mbal.Locked)
 				}
-
-				tw.Write(row)
+				wallets = append(wallets, wallet)
 			}
 		}
 
 		if !cctx.Bool("addr-only") {
-			return tw.Flush(os.Stdout)
+
+			if cctx.Bool("json") {
+				// get a new list of wallets with json keys instead of tablewriter keys
+				var jsonWallets []map[string]interface{}
+				for _, wallet := range wallets {
+					jsonWallet := make(map[string]interface{})
+					for k, v := range wallet {
+						jsonWallet[tableKeysToJsonKeys[k]] = v
+					}
+					jsonWallets = append(jsonWallets, jsonWallet)
+				}
+				// then return this!
+				return cmd.PrintJson(jsonWallets)
+			} else {
+				// Init the tablewriter's columns
+				tw := tablewriter.New(
```

Comment:
It would have been nice not to have add all of the tableWriter.Col(<someKey) manually during tablewriter's creation. Instead, it would be nice if we could instantiate an empty tablewriter using something like tablewrite.New(), and then, dynamically populate the columns with the keys within the for loop below.
That would have been more maintainable.